src/config/flat/typescript.ts

```diff
@@ -30,6 +30,8 @@ export default {
       node: {
```

Comment:
How about removing `node` resolver?
`typescript` resolver should be enough for default `typescript` configs

Reply:
Sure! I personally only uses typescript resolver as well.